src/sources/dataModel/cordraSchema.json

```diff
@@ -0,0 +1,243 @@
+            "type": {
+              "autoGeneratedField": "handle"
+            }
+          }
+        },
+        "ods:type": {
+          "type": "string",
+          "title": "Object Type",
+          "enum": [
+            "taxonomicService"
+          ]
+        },
+        "dcterms:license": {
+          "type": "string",
+          "title": "License"
+        },
+        "cetaf:state": {
+          "type": "string",
+          "title": "State",
+          "enum": [
+            "draft",
+            "published"
+          ]
+        },
+        "erp:languageAvailabilities": {
+          "type": "array",
+          "title": "Languages supported",
+          "items": {
+            "type": "string"
+          }
+        },
+        "erp:name": {
+          "type": "string",
+          "maxLength": 50,
+          "title": "Name",
+          "cordra": {
+            "preview": {
+              "showInPreview": true,
+              "isPrimary": true
+            }
+          }
+        },
+        "erp:description": {
+          "type": "string",
+          "format": "textarea",
+          "maxLength": 2048,
+          "title": "Description",
+          "cordra": {
+            "preview": {
+              "showInPreview": true,
+              "excludeTitle": true
+            }
+          }
+        },
+        "erp:tagLine": {
+          "type": "string",
+          "maxLength": 100,
+          "title": "Tag Line"
+        },
+        "erp:logo": {
+          "type": "string",
+          "title": "Logo URL"
+        },
+        "cetaf:taxonomicServiceType": {
+          "type": "string",
+          "title": "Taxonomic Service Type",
+          "enum": [
+            "taxonomicService",
+            "referenceCollection"
+          ]
+        },
+        "erp:category": {
+          "type": "string",
+          "title": "Category",
+          "enum": [
+            "referenceCollection"
+          ]
+        },
+        "erp:EOSC_serviceId": {
+          "type": "string",
+          "title": "EOSC Service ID"
+        },
+        "erp:version": {
+          "type": "string",
+          "title": "Version of taxonomic service"
+        },
+        "erp:lastUpdate": {
+          "type": "string",
+          "title": "Last updated"
+        },
+        "cetaf:taxonomicScope": {
+          "type": "array",
+          "title": "Taxonomic Scope",
+          "items": {
+            "type": "string"
+          }
+        },
+        "cetaf:qualityScore": {
+          "type": "number",
+          "title": "Quality Score"
+        },
+        "cetaf:reviewComments": {
+          "type": "string",
+          "format": "textarea",
+          "maxLength": 2048,
+          "title": "Review Comments"
+        },
+        "erp:helpdeskEmail": {
+          "type": "string",
+          "title": "Helpdesk Email"
+        },
+        "erp:helpdeskPage": {
+          "type": "string",
+          "title": "Helpdesk Page"
+        },
+        "cetaf:agents": {
```

Comment:
You could also follow the DiSSCo style rules. Then this would become `hasAgent`: [{`@type`: `cetaf:Agent`}] 
I know that this has been approved but would be nice to get a generic standards

Reply:
Sounds good, will change that when I have time for it in an upcoming PR